landscape/categories.md

```diff
@@ -6,7 +6,9 @@ sub-categories. The remaining text is a description of each sub-category.
 
 - **App definition and development**
   - _Static Code Analysis_
-    - Inspecting code for OWASP vulnerabilities, pen testing, etc
+    - Inspecting code for vulnerabilities, pen testing, etc
```

Comment:
I think that OWASP was intended to be an example, but agree with dropping it since it can read as overly specific.  Thanks for reviewing this!